README.md

```diff
@@ -91,12 +91,14 @@ The task folder in this repo holds an existing site, including multiple html fil
 
 ### Serving the home page
 
-1. Create a ```home.hbs``` in the views directory
-1. Navigate to public/index.html. Copy all the contents to ```home.hbs```.
+1. In the views directory, create a file called `home.hbs`.
```

Comment:
Can you:
- move this to 3? It feels weird creating the file, then not using it until step 4.
- use full path names. I imagine students will feel overwhelmed with the file structure in the repo so any extra info to help them find things is 👌 
- fix the numbers on this, it renders find in markdown but is confusing to read raw.